vnext/DeforkingPatches/ReactCommon/yoga/yoga/YGStyle.h

```diff
@@ -0,0 +1,388 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the LICENSE
+ * file in the root directory of this source tree.
+ */
+#pragma once
+#include <algorithm>
+#include <array>
+#include <bitset>
+#include <cstdint>
+#include <type_traits>
+#include "CompactValue.h"
+#include "YGEnums.h"
+#include "YGFloatOptional.h"
+#include "Yoga-internal.h"
+#include "Yoga.h"
+
+// Until we sync past this:
+// https://github.com/facebook/react-native/commit/a7e4ce0f0d83b6162d77db7d680e838666f6d938#diff-140c478f1096be49a1b121931a7680ee
+// We have to turn off the bitfield usage for MSVC builds
+#if defined(_MSC_VER) && !defined(ENUM_BITFIELDS_NOT_SUPPORTED)
```

Comment:
The added `&& !defined(ENUM_BITFIELDS_NOT_SUPPORTED)` is the only change here. We can remove it immediately after switching to upstream React Native.

Reply:
We could also just remove this entire block if we wanted to. Doesn't matter too much since this won't be around long.

Reply:
We should create a note in the project board for RN61?

---
In reply to: [372135279](https://github.com/microsoft/react-native-windows/pull/3976#discussion_r372135279) [](ancestors = 372135279)